package/src/components/Channel/Channel.tsx

```diff
@@ -494,7 +493,7 @@ const ChannelWithContext = <
     AudioAttachment = AudioAttachmentDefault,
     AudioAttachmentUploadPreview = AudioAttachmentDefault,
     AudioRecorder = AudioRecorderDefault,
-    audioRecordingEnabled = isAudioRecorderAvailable(),
+    audioRecordingEnabled = false,
```

Comment:
This one is a breaking change. I'd leave this value as it was previously.

Reply:
Nah, in reality, we mistakenly introduced a breaking change here and the audio button was visible no matter what. This PR really fixes that.

Reply:
yes, the PR fixes the original issue but, here the change of this prop creates the opposite problem.
In case when someone has `expo-av` installed, without `<Channel audioRecordingEnabled={true} />` the record button will be removed, although that wasn't the case.

Reply:
Well expo-av is also used for video playing capabilities in the sdk. So, having expo-av doesn't mean an integrator would want async audio necessarily.

Reply:
ah, I see. Thanks for the clarification.